README.md

```diff
@@ -101,7 +102,7 @@ Can also be set as an attribute on the element, e.g. `<virtual-list layout="hori
 
 This re-renders all of the currently-displayed items, updating them from their source data using `updateChild`.
 
-This can be useful for if you mutate the `items` array, or elements in it, instead of setting the `items` property to a new value. Also see [the example below](#data-manipulation-using-requestreset).
+This can be useful when you mutate data without changing the amount of `items`. Also see [the example below](#data-manipulation-using-requestreset).

```

Comment:
We'll probably want a bigger documentation and naming overhaul for requestReset(), since it's now very very important. But we can do that as a follow-up; I'm eager to help.